src/components/User/GetVpDropdown.tsx

```diff
@@ -0,0 +1,47 @@
+import { Dropdown } from 'decentraland-ui/dist/components/Dropdown/Dropdown'
+import { useMobileMediaQuery } from 'decentraland-ui/dist/components/Media/Media'
+
+import useFormatMessage from '../../hooks/useFormatMessage'
+
+import './GetVpDropdown.css'
+
+export default function GetVpDropdown() {
+  const isMobile = useMobileMediaQuery()
+  const t = useFormatMessage()
+
+  const buttonText = isMobile ? t('page.profile.get_vp_dropdown.get_vp') : t('page.profile.get_vp_dropdown.get_more_vp')
+
+  return (
+    <Dropdown className="GetVpDropdown" text={buttonText} button direction="left">
+      <Dropdown.Menu>
+        <Dropdown.Item
+          as="a"
+          href="https://decentraland.org/marketplace/lands?assetType=nft&section=land&vendor=decentraland&page=1&sortBy=newest&onlyOnSale=true"
```

Comment:
Shouldn't the links be in environment variables? Or in a constants file at least

Reply:
I'd move it to a constants file.